rbi/annotations/parse-cron.rbi

```diff
@@ -2,8 +2,6 @@
 
 # Parses cron expressions and computes the next occurence of the "job"
 class CronParser
-  extend T::Generic
```

Comment:
See https://github.com/siebertm/parse-cron/blob/master/lib/cron_parser.rb (not generic)